Automattic/jetpack
# Social: Display the share counter.

We are going to be displaying the number of shares made in the past 30 days. This PR is a WIP and currently displays the share counter by calling the API, however it does that on the old design we had. I will work on updating the design here, next. 





#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the Jetpack Social admin page and observe that the share count shown to you is accurate. 
* Test for responsiveness by viewing the page in different screen resolutions. 




https://user-images.githubusercontent.com/6594561/175268756-91e658aa-87f1-461b-999b-bb71b0530cc0.mov
